stock-managment/src/main/java/org/dhis2/android/rtsm/ui/home/screens/components/MainContent.kt

```diff
@@ -70,6 +76,7 @@ fun MainContent(
     val weightValueArrow = if (backdropState.isRevealed) 0.10f else 0.05f
     val weightValueArrowStatus = backdropState.isRevealed
     val focusManager = LocalFocusManager.current
+    val search = viewModel.scanText.collectAsState().value
```

Comment:
is the manual search being affected by this?

Reply:
Yes. Because we get value from here, and we just set onValueChanged